packages/block-editor/src/components/global-styles/typography-panel.js

```diff
@@ -104,7 +104,7 @@ function getUniqueFontSizesBySlug( settings ) {
 	const fontSizes = settings?.typography?.fontSizes;
 	const mergedFontSizes = fontSizes ? mergeOrigins( fontSizes ) : [];
 	const uniqueSizes = [];
-	for ( const currentSize of mergedFontSizes ) {
+	for ( const currentSize of mergedFontSizes.reverse() ) {
```

Comment:
I wouldn't use `.reverse()` like this. It mutates the array in-place. Having read through `mergeOrigins()`, it looks like it returns a new array, so it wouldn't have side-effects here, but it would be better to attach the `.reverse()` to `mergeOrigins( fontSizes ).reverse()` so the relationship between them is more clear.

Alternatively, if you don't want to mutate the original array, `[].toReversed()` has good browser support these days at the cost of more memory usage.